components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -243,7 +253,6 @@ InfoListItem.defaultProps = {
     chevron: false,
     dense: false,
     hidePadding: false,
-    onClick: (): void => {},
```

Comment:
Why remove the default value?

Reply:
The change you are refering to above used an if statement... which no longer needs a default onClick. 

And when does the default assignment happen? This: 
``` typescript
cursor: onClick ? 'pointer' : 'default',
```
in `getWrapperStyle` will always set the cursor to pointer if there is a default value set in defaultProps?